### DISCUSSION TITLE
Unable to sign in to Casdoor with default credentials

### DISCUSSION THREAD

**Discussion by AmythD** [casdoor/casdoor#3440] [Q&A] [answered]
Hello,

I am trying to get Casdoor running using a local sqlite DB, but when trying to login using default credentials, I am getting below errors:

I am not seeing a lot of errors on console, any ideas on what could be wrong?

**Configuration : app.conf**

```
appname = casdoor
httpport = 8000
runmode = dev
copyrequestbody = true
driverName = sqlite
dataSourceName = "file:casdoor.db?cache=shared"
dbName = casdoor
tableNamePrefix =
showSql = false
redisEndpoint =
defaultStorageProvider =
isCloudIntranet = false
authState = "casdoor"
socks5Proxy = "127.0.0.1:10808"
verificationCodeTimeout = 10
initScore = 0
logPostOnly = true
isUsernameLowered = false
origin =
originFrontend =
staticBaseUrl = "https://cdn.casbin.org"
isDemoMode = false
batchSize = 100
enableErrorMask = false
enableGzip = true
inactiveTimeoutMinutes =
ldapServerPort = 389
ldapsCertId = ""
ldapsServerPort = 636
radiusServerPort = 1812
radiusSecret = "secret"
quota = {"organization": -1, "user": -1, "application": -1, "provider": -1}
logConfig = {"filename": "logs/casdoor.log", "maxdays":99999, "perm":"0770"}
initDataNewOnly = false
initDataFile = "./init_data.json"
frontendBaseDir = "../cc_0"
```

**Browser:**

![screenshot 524](https://github.com/user-attachments/assets/e17ddb1a-b8bd-4b7c-aea7-b4e7afede902)

**Backend:**

![screenshot 525](https://github.com/user-attachments/assets/2510a4d1-7eb9-4930-8ce7-a4c17d0f68cf)

**Frontend:**

![screenshot 526](https://github.com/user-attachments/assets/1201b224-97a9-4e0b-abab-bda2c997fa66)

Regards,
Amith

**Comment by hsluoyz** [comment, marked as answer]
@AmythD I don't use sqlite DB

**Comment by Hassanzadeh-sd** [comment]
user : Admin
pass : 123
